src/ontology/mondo-edit.obo

```diff
@@ -179274,7 +179274,6 @@ relationship: excluded_subClassOf MONDO:0017753 {source="Orphanet:79095", source
 relationship: has_characteristic MONDO:0021140 {source="https://orcid.org/0000-0002-4142-7153"} ! congenital
 relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/451 {source="MONDO:mim2gene_medgen"} ! AMACR
 property_value: confidence "5.666666666666666" xsd:double
-property_value: excluded_subClassOf "'rare hereditary metabolic disease with peripheral neuropathy'" xsd:string {source="Orphanet:79095"}
```

Comment:
@sabrinatoro I see this deleted here but it is present above, so this looks okay